ibc-apps/ics721-nft-transfer/types/src/data.rs

```diff
@@ -294,9 +289,9 @@ mod tests {
     #[test]
     fn test_borsh_roundtrip() {
         fn borsh_roundtrip(data_value: DataValue) {
-            use borsh::{BorshDeserialize, BorshSerialize};
+            use borsh::BorshDeserialize;
 
-            let data_value_bytes = data_value.try_to_vec().unwrap();
+            let data_value_bytes = borsh::to_vec(&data_value).unwrap();
```

Comment:
why is the trait import removed and `borsh::to_vec` is used?

Reply:
`BorshSerialize::try_to_vec` looks like it was removed in v1; at least, I couldn't find it in the `borsh` docs.

Reply:
I see. Yea, looking into `borsh::to_vec` method - it just calls `value.serialize`. In that case, can we also replace `DataValue::try_from_slice` with [`borsh::from_slice`](https://docs.rs/borsh/latest/borsh/de/fn.from_slice.html) too?

PS. This should also remove the local imports then.